ideascube/search/utils.py

```diff
@@ -1,18 +1,29 @@
 import struct
 
-from django.db import connection
+from django.db import connections
 
 
 def create_index_table(force=True):
-    cursor = connection.cursor()
-    cursor.execute("SELECT count(*) FROM sqlite_master "
-                   "WHERE type='table' AND name='idx';")
-    count = cursor.fetchone()[0]
+    cursor_transient = connections['transient'].cursor()
+    cursor_transient.execute("SELECT count(*) FROM sqlite_master "
+                             "WHERE type='table' AND name='idx';")
+    count = cursor_transient.fetchone()[0]
     if not count or force:
-        cursor.execute("DROP TABLE IF EXISTS idx")
-        cursor.execute("CREATE VIRTUAL TABLE idx using "
-                       "FTS4(id, model, model_id, public, text)")
+        cursor_transient.execute("DROP TABLE IF EXISTS idx")
+        cursor_transient.execute("CREATE VIRTUAL TABLE idx using "
+                                 "FTS4(id, model, model_id, public, text)")
```

Comment:
Is there a reason we can't use Django here, to create the table, since it maps to the `Search` model?